### ISSUE TITLE
The dataset split for link prediction

### ISSUE THREAD

**Issue by lei-liu1** [LuckyTiger123/GraphTCM#1]
Hello,

Thanks for sharing the codes. 
I would like to know the dataset split ratio for the link prediction task, but I couldn't find it in the paper or the codes. Could you please share the split ratio of the dataset used for the results reported in the paper?

Best Regards,
Lei

**Comment by LuckyTiger123** [comment]
Thanks for your interest in our work. In link prediction tasks on Cora, CiteSeer, and PubMed, we randomly selected 5% as the validation set and 10% as the test set. We conducted link prediction experiments following the procedure outlined in https://github.com/pyg-team/pytorch_geometric/blob/master/examples/link_pred.py. We hope our response can address your issue.

**Comment by lei-liu1** [comment]
Thanks!
